README.md

```diff
@@ -236,6 +236,25 @@ Linux wheels are built in the [`manylinux1` docker images](https://github.com/py
 - The project directory is mounted in the running Docker instance as `/project`, the output directory for the wheels as `/output`. In general, this is handled transparently by `cibuildwheel`. For a more finegrained level of control however, the root of the host file system is mounted as `/host`, allowing for example to access shared files, caches, etc. on the host file system.  Note that this is not available on CircleCI due to their Docker policies.
 - Alternative dockers images can be specified with the `CIBW_MANYLINUX1_X86_64_IMAGE` and `CIBW_MANYLINUX1_I686_IMAGE` options to allow for a custom, preconfigured build environment for the Linux builds. See [below](#options) for more details.
 
+Moden C++ standards
+-------------------
+Python 2.7 use `register` keyword which is prohibited in c++17 standard. 
+
+Linux
+=====
+For `manulinux1` images only c++11 standards is supported. c++14 and c++17 will be supported with manylinux2010 image
+
+MacOS
+=====
+To get c++11 and c++14 upport need to set `MACOSX_DEPLOYMENT_TARGET` variable to `10.9` value
```

Comment:
```suggestion
To get c++11 and c++14 support need to set `MACOSX_DEPLOYMENT_TARGET` variable to `10.9` value
```

Do newer macos versions work as well? Or just `10.9`?

Reply:
10.9 is minimal version of macos on which package will work.